stable/columbus/values.yaml

```diff
@@ -0,0 +1,62 @@
+app:
+  image:
+    repository: odpf/columbus
+    pullPolicy: Always
+    tag: latest
```

Comment:
Let's lock the version here? What if a new version has something that requires some other customisation here.

Reply:
We can use appVersion field by default from `Chart.yaml` ?

Reply:
Hardcoded tag to `0.1.8` (latest tag).


> We can use appVersion field by default from Chart.yaml ?

@rohilsurana We might need to use .tpl for this since `values.yaml` is not templatized.